server/lib/nl/utils.py

```diff
@@ -525,28 +525,26 @@ def get_immediate_parent_places(
     main_place_dcid: str, parent_place_type: str,
     counters: ctr.Counters) -> List[detection.Place]:
   start = time.time()
-  resp = dc.property_values_v1([main_place_dcid], 'containedInPlace')
+  resp = dc.property_values([main_place_dcid], 'containedInPlace')
   counters.timeit('get_immediate_parent_places', start)
   results = []
-  for item in resp.get('data', []):
-    if item.get('node', '') != main_place_dcid:
+  for value in resp['data'][main_place_dcid].get('arcs', {}).get(
```

Comment:
nit: to avoid key access should we do:  `resp.get('data', {}).get(main_place_dcid, {}).get(...)` ?

Reply:
'data' and <place> always exist in the response, but changed to get() to make it more robust